### ISSUE TITLE
search while typing -- needs cue?

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-project#809]
Tester JB says: 
In addition a "search while typing" would be ideal (e.g. if I type "hom" that the system suggests homo sapiens and whatever else starts/contains "hom").

This suggests that it isn't obvious that you can search by typing. Perhaps a 'start typing' cue in the empty field...?

**Comment by jpburckert** [comment]
Yes that would help - the BioPortal Access unfortunately takes almost a minute on my end.

**Comment by willrett** [comment]
Related to #797
